hazelcast-spring/src/main/resources/hazelcast-spring-3.11.xsd

```diff
@@ -3339,14 +3342,6 @@
         </xs:complexContent>
     </xs:complexType>
 
-    <xs:complexType name="aws-client">
-        <xs:complexContent>
-            <xs:extension base="aws">
-                <xs:attribute name="inside-aws" use="optional" type="parameterized-boolean" default="false"/>
```

Comment:
Can the user still use parametrized values? I guess so since this only enforces the pattern and doesn't actually do any replacement.

Reply:
Yeah, it's gonna work the same, here's just the XSD validation and the XSD validation actually becomes more open (it accepts any string, so also parameterized boolean).